database/engine/README.md

```diff
@@ -59,21 +131,23 @@ Netdata metric values per legacy database engine instance (see [details on the l
 
 ### Streaming metrics to the database engine
```

Comment:
Since this is the only mode we have now, I don't see much use for this section. 

I'm thinking that maybe we should move together every option that has changed in a "Older versions/Backwards compatibility" section, as they are now in various places.